fix(ci): build fails upon PR merge

# Bug Report

## Description
The `build` workflow fails upon PR merge. 

## Steps to Reproduce

1. Create a PR and merge it with main
2. `build` workflow fails

## Actual Behavior
`build` fails owing to unclosed tags in `galleries/index.html`.

## Expected Behavior
The `build` workflow should execute without issues when all aspects of the change are in order.

## Screenshots
Following is the workflow failure error report:
```
Run yarn format
yarn run v1.22.22
$ prettier --write .
.eslintrc.json 37ms
.github/workflows/build.yml 27ms
.github/workflows/commit-message-lint.yml 5ms
.github/workflows/release.yml 7ms
.prettierrc.json [4](https://github.com/xability/maidr/actions/runs/10587976139/job/29339717914#step:5:5)ms
.vscode/extensions.json 7ms
.vscode/settings.json 6ms
.vscode/tasks.json 7ms
commitlint.config.js 11ms
galleries/bar_plot.html 100ms
galleries/boxplot_horizontal.html 103ms
galleries/boxplot_vertical.html 66ms
galleries/dodged_bar.html 142ms
galleries/heatmap.html [5](https://github.com/xability/maidr/actions/runs/10587976139/job/29339717914#step:5:6)5ms
galleries/histogram.html [6](https://github.com/xability/maidr/actions/runs/10587976139/job/29339717914#step:5:7)9ms

Error:  galleries/index.html: SyntaxError: Unexpected closing tag "body". It may happen when the tag has already been closed by another tag. For more info see https://www.w3.org/TR/html5/syntax.html#closing-elements-that-have-implied-end-tags (113:3)
Error:    111 | </p>
Error:    112 |     <script src="/src/js/init.js"></script>
Error:  > 113 |   </body>
Error:        |   ^^^^^^^
Error:    114 | </html>
Error:    115 |
galleries/lineplot.html 126ms
galleries/scatterplot.html 119ms
galleries/stacked_normalized.html [7](https://github.com/xability/maidr/actions/runs/10587976139/job/29339717914#step:5:8)3ms
galleries/stacked.html 67ms
galleries/svglite_test.html 51ms
gulpfile.js 15ms
index.html 7ms
jsconfig.json 2ms
jsdoc.json 3ms
package.json 3ms
src/css/styles.css 45ms
src/js/__tests__/audio.test.js [8](https://github.com/xability/maidr/actions/runs/10587976139/job/29339717914#step:5:9)ms
src/js/__tests__/constants.test.js 74ms
src/js/audio.js 56ms
src/js/barplot.js 28ms
src/js/boxplot.js 5[9](https://github.com/xability/maidr/actions/runs/10587976139/job/29339717914#step:5:10)ms
src/js/constants.js 154ms
src/js/controls.js 207ms
src/js/display.js 66ms
src/js/heatmap.js 24ms
src/js/histogram.js [10](https://github.com/xability/maidr/actions/runs/10587976139/job/29339717914#step:5:11)ms
src/js/init.js 25ms
src/js/lineplot.js 13ms
src/js/scatterplot.js 46ms
src/js/segmented.js 16ms
user_study_2/_task1_lineplot.html 253ms
user_study_2/_task2_boxplot_horizontal.html 41ms
user_study_2/data/boxplot/boxplot_maidr_data.json 5ms
user_study_2/data/boxplot/boxplot.html 42ms
user_study_2/data/lineplot/lineplot_alltime_maidr_data.json [14](https://github.com/xability/maidr/actions/runs/10587976139/job/29339717914#step:5:15)ms
user_study_2/data/lineplot/lineplot.html 197ms
user_study_2/intro.html 3ms
user_study_2/task1_boxplot_horizontal.html 29ms
user_study_2/task2_lineplot.html 79ms
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 2.
```

## Environment
- Operating System: MacOS Sonoma 14.6.1
- Browser: Google Chrome
- Version: 2.14.6

--- a/galleries/index.html
+++ b/galleries/index.html
@@ -15,18 +15,32 @@
 
   <body>
     <div class="container mt-3">
-
       <h1>Examples of Multimodal Data Representation</h1>
       <p>
-        The following examples demonstrate how maidr system works. Once you open any of the links below, press tab key (or click the plot area) to focus on the plot. Then, you can use the following keys to interact with the plot:
+        The following examples demonstrate how maidr system works. Once you open
+        any of the links below, press tab key (or click the plot area) to focus
+        on the plot. Then, you can use the following keys to interact with the
+        plot:
       </p>
       <ul>
         <li>Arrow keys: move the data point</li>
-        <li>ctrl+LeftArrow / ctrl+Right+Arrow (on Windows and Linux): autoplay data points either to left or to right</li>
-        <li>cmd+LeftArrow / cmd+Right+Arrow (on Mac): autoplay data points either to left or to right</li>
-        <li>PageUp / PageDown: switch layers (currently supported in scatterplots only)</li>
+        <li>
+          ctrl+LeftArrow / ctrl+Right+Arrow (on Windows and Linux): autoplay
+          data points either to left or to right
+        </li>
+        <li>
+          cmd+LeftArrow / cmd+Right+Arrow (on Mac): autoplay data points either
+          to left or to right
+        </li>
+        <li>
+          PageUp / PageDown: switch layers (currently supported in scatterplots
+          only)
+        </li>
         <li>B: toggle braille</li>
-              <li>T: toggle text announcement (it cycles through "terse", "verbose", and "off")</li>
+        <li>
+          T: toggle text announcement (it cycles through "terse", "verbose", and
+          "off")
+        </li>
         <li>S: toggle sonification (spatial sound)</li>
         <li>H: open Help and settings menu</li>
         <li>ctrl+shift+/ (on Windows and Linux): toggle AI/LLM chat</li>
@@ -36,19 +50,13 @@
       <h2>Bar Plot</h2>
       <ul>
         <li>
-          <a target="_blank" href="bar_plot.html"
-            >Simple Bar Plot</a
-          >
+          <a target="_blank" href="bar_plot.html">Simple Bar Plot</a>
         </li>
         <li>
-          <a target="_blank" href="dodged_bar.html"
-            >Dodged Bar Plot</a
-          >
+          <a target="_blank" href="dodged_bar.html">Dodged Bar Plot</a>
         </li>
         <li>
-          <a target="_blank" href="stacked.html"
-            >Stacked Bar Plot</a
-          >
+          <a target="_blank" href="stacked.html">Stacked Bar Plot</a>
         </li>
         <li>
           <a target="_blank" href="stacked_normalized.html"
@@ -59,17 +67,13 @@
       <h2>Histogram</h2>
       <ul>
         <li>
-          <a target="_blank" href="histogram.html"
-            >Histogram</a
-          >
+          <a target="_blank" href="histogram.html">Histogram</a>
         </li>
       </ul>
       <h2>Line Plot</h2>
       <ul>
         <li>
-          <a target="_blank" href="lineplot.html"
-            >Line Plot</a
-          >
+          <a target="_blank" href="lineplot.html">Line Plot</a>
         </li>
       </ul>
 
@@ -95,20 +99,30 @@
       <h2>Scatter Plot</h2>
       <ul>
         <li>
-          <a target="_blank" href="scatterplot.html"
-            >Scatter Plot</a
-          >
+          <a target="_blank" href="scatterplot.html">Scatter Plot</a>
         </li>
       </ul>
-<h1>Contact</h1>
+      <h1>Contact</h1>
 
-<p>
-  If you have any questions or suggestions, please contact us at
-  <a href="mailto:xability-lab@illinois.edu">xability-lab@illinois.edu</a>.
-</p>
-<p>
-  Developed by <a href="https://xabilitylab.ischool.illinois.edu" target="_blank">(x)Ability Design Lab</a> in the School of Information Sciences at the University of Illinois Urbana-Champaign. Directed by <a href="https://ischool.illinois.edu/people/jooyoung-seo" target="_blank">Dr. JooYoung Seo</a>.
-</p>
-    <script src="/src/js/init.js"></script>
+      <p>
+        If you have any questions or suggestions, please contact us at
+        <a href="mailto:xability-lab@illinois.edu">xability-lab@illinois.edu</a
+        >.
+      </p>
+      <p>
+        Developed by
+        <a href="https://xabilitylab.ischool.illinois.edu" target="_blank"
+          >(x)Ability Design Lab</a
+        >
+        in the School of Information Sciences at the University of Illinois
+        Urbana-Champaign. Directed by
+        <a
+          href="https://ischool.illinois.edu/people/jooyoung-seo"
+          target="_blank"
+          >Dr. JooYoung Seo</a
+        >.
+      </p>
+      <script src="/src/js/init.js"></script>
+    </div>
   </body>
 </html>

